src/Facebook/InstantArticles/Transformer/Getters/ElementGetter.php

```diff
@@ -53,7 +53,7 @@ public function withSelector($selector)
     public function get($node)
     {
         $elements = self::findAll($node, $this->selector);
-        if (!empty($elements)) {
+        if (!empty($elements) and property_exists($elements, 'length') and $elements->length !== 0) {
```

Comment:
Can you please replace "and" with "&&"?
"and" isn't being used on the sdk code.

Thanks for your help!